cbirrt2/catkin.cmake

```diff
@@ -41,3 +42,12 @@ target_link_libraries("${PROJECT_NAME}_plugin"
     ${Boost_LIBRARIES}
     ${catkin_LIBRARIES}
 )
+
+catkin_add_gtest(test_wam7 test/test_wam7.cpp)
+target_link_libraries(test_wam7 ${OpenRAVE_LIBRARIES} ${OpenRAVE_CORE_LIBRARIES})
+
+catkin_add_gtest(test_wam7_con test/test_wam7_con.cpp TaskSpaceRegion.cpp)
+target_link_libraries(test_wam7_con ${OpenRAVE_LIBRARIES} ${OpenRAVE_CORE_LIBRARIES})
```

Comment:
What is the difference between `test_wam7` and `test_wam7_con`? If the latter is just tests for TSR sampling, can you give it a more descriptive name like `test_tsr`?